augur/tasks/start_tasks.py

```diff
@@ -175,7 +175,10 @@ def core_task_success_util_gen(repo_git, full_collection):
     primary_enabled_phases.append(core_task_success_util_gen)
     primary_gitlab_enabled_phases.append(core_task_success_util_gen)
 
-    primary_request = CollectionRequest("core",primary_enabled_phases,max_repo=40, days_until_collect_again=15, gitlab_phases=primary_gitlab_enabled_phases)
+    # Get core collection interval from config
+    tasks_config = get_section("Tasks")
+    core_interval = tasks_config.get('core_collection_interval', 15)
+    primary_request = CollectionRequest("core",primary_enabled_phases,max_repo=40, days_until_collect_again=core_interval, gitlab_phases=primary_gitlab_enabled_phases)
```

Comment:
the days_until_collect_again parameter is unused. I think it should use the parameter and the config value should be injected at the point where build_primary_repo_collect_request is called (same for all these other similar functions)